sklearn/cluster/hierarchical.py

```diff
@@ -902,7 +920,7 @@ class FeatureAgglomeration(AgglomerativeClustering, AgglomerationTransform):
     n_leaves_ : int
         Number of leaves in the hierarchical tree.
 
-    n_components_ : int
```

Comment:
You also need to deprecate this one like you did for `AgglomerativeClustering`

Reply:
My bad ! This is fixed in commit [#17aded9](https://github.com/scikit-learn/scikit-learn/pull/13427/commits/17aded92257d5ef70fa2cc07b990dcf77cc5806d)